config/local.js

```diff
@@ -0,0 +1,51 @@
+const BINPATH = './node_modules/nightwatch/bin/'
```

Comment:
Yes, the npm module will just download them and put them in the correct place - still need to point nightwatch to where they are.